settings.php

```diff
@@ -0,0 +1,36 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Link to plugin generator.
+ *
+ * @package    tool_pluginkenobi
+ * @copyright  2016 Alexandru Elisei
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+defined('MOODLE_INTERNAL') || die;
+
+// Entry in Site administration -> Development -> Generate plugin skeleton.
+if ($hassiteconfig) {
+    $ADMIN->add(
```

Comment:
Can we have a global setting for this tool that would allow to define default values that are likely to be re-used every time. Typically I would like to fill my name and copyright values so that I do not need to retype them all the time.